### PR TITLE
Dashboard children/show children

### PR DESCRIPTION
**Pull request by heyitsmaimai**

### CODE REVIEW

File: test/unit/specs/dashboard/cd-dashboard-children.spec.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+
+  describe('computed', () => {
+    describe('computed.children', () => {
+      it('should return a list containing the current user if isSingle', () => {
```
Comment (Wardormeur) [inline test/unit/specs/dashboard/cd-dashboard-children.spec.js]: Change tests names, isSingle doesn't exists :)

File: test/unit/specs/dashboard/cd-dashboard-children.spec.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    it('should add a default child if the user is having no child and is not single', async () => {
+      const vm = vueUnitHelper(DashboardChildrenComponentWithMocks);
+      vm.loadCurrentUser = sinon.stub().resolves();
+      vm.loadProfile = sinon.stub().resolves();
+      vm.loadChildren = sinon.stub().resolves();
+      vm.currentUser = null;
+      vm.userProfile = {};
+      vm.userChildren = [];
+
+      await vm.$lifecycleMethods.created();
```
Comment (Wardormeur) [inline test/unit/specs/dashboard/cd-dashboard-children.spec.js]: Missing test? What do we assert here?

**Comment by Wardormeur** [comment]
Also : translations !!1

File: test/unit/specs/dashboard/cd-dashboard-children.spec.js (hunk 2 above)
Comment (heyitsmaimai) [reply test/unit/specs/dashboard/cd-dashboard-children.spec.js]: errr is it that it actually loads the children if they have kids and doesn't if none?

Comment (Wardormeur) [reply test/unit/specs/dashboard/cd-dashboard-children.spec.js]: It doesn't test anything, it stubs data, but there is no "expect".
You would expect here (and the test name is wrong) that each of the component methods are being called to load the data

File: src/dashboard/cd-dashboard-children.vue [hunk 3]
```diff
@@ -1,16 +1,56 @@
 <template>
   <div class="column">
     <div class="cd-dashboard-children">
-      <h1 class="cd-dashboard-children__header">My Kids</h1>
+      <h1 class="cd-dashboard-children__header">{{ $t('My Children') }}</h1>
+      <div class="cd-dashboard-children__child" v-for="child in children">
+        <h3>{{ child.firstName }} {{ child.lastName }}</h3>
```
Comment (DanielBrierton) [inline src/dashboard/cd-dashboard-children.vue]: Oh, actually... I think we have `child.name` available that has them merged already.

File: src/dashboard/cd-dashboard-children.vue [hunk 4]
```diff
@@ -24,9 +64,16 @@
     margin-left: auto;
     min-height: 100%;
     display:flex;
+    flex-direction: column;
```
Comment (DanielBrierton) [inline src/dashboard/cd-dashboard-children.vue]: Sorry for third random review, just noticed this in a different PR. 😅  Seems like there's no need for this being flex now that it's column.

Would also be curious if `min-height` is needed here too. I think it is for the moment, and I think I know why, but we should be able to fix that to simplify things. Don't worry about it for now. I'll probably do an overall audit of the CSS anyway and see where we can simplify things. We'll see. 😛

Comment (heyitsmaimai) [reply src/dashboard/cd-dashboard-children.vue]: oh the flex column is for the elements inside the children to be displayed stacked 
-
-
-
^ like that

Comment (DanielBrierton) [reply src/dashboard/cd-dashboard-children.vue]: Column stacking is the default behaviour for block elements, which is all that's used inside. It's no biggie, but it should work without flex unless I'm missing something.

Comment (heyitsmaimai) [reply src/dashboard/cd-dashboard-children.vue]: yeah all good. just fixed that there

**Pull request merged**